src/Workspaces/VisualBasic/Portable/Formatting/Engine/Trivia/TriviaDataFactory.ComplexTrivia.vb

```diff
@@ -81,6 +81,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
 
                 Debug.Assert(Me.SecondTokenIsFirstTokenOnLine OrElse beginningOfNewLine)
 
+                If Me.OptionSet.GetOption(FormattingOptions.UseTabs, LanguageNames.VisualBasic) Then
```

Comment:
That's entirely possible but as I said in my original comment, the fix is copied over from `Microsoft.CodeAnalysis.CSharp.Formatting.TriviaDataFactory.ComplexTrivia.ShouldFormat` and I believe the person writing those lines had better understanding of the code than I so I reused them.

Therefore the solution I am providing is entirely up to performance standards set up by current c# complex trivia formatter and ultimately roslyn itself. To improve the performance, feature request should be started.